### PR TITLE
 Create p9e.de.xml 

### PR DESCRIPTION
**Pull request by ihave100openPRs**

### CODE REVIEW

File: src/chrome/content/rules/p9e.de.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="p9e.de">
+	<target host="p9e.de" />
+	<target host="www.p9e.de" />
```
Comment (J0WI) [inline src/chrome/content/rules/p9e.de.xml]: `git.p9e.de`

File: src/chrome/content/rules/searx.fi.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="searx.fi">
+	<target host="searx.fi" />
+	<target host="www.searx.fi" />
```
Comment (J0WI) [inline src/chrome/content/rules/searx.fi.xml]: I get a `401` on both targets.

Comment (ihave100openPRs) [reply src/chrome/content/rules/searx.fi.xml]: @J0WI It was working before without authentication, I asked its operator and he said that he put it behind a password: https://github.com/asciimoo/searx/issues/1072#issuecomment-347153737

Do you think I should remove it nevertheless?

Comment (J0WI) [reply src/chrome/content/rules/searx.fi.xml]: I wounder why the tests are passing since we are not able to test this host.
I would remove it, yes.

**Comment by ihave100openPRs** [comment]
@J0WI Ok, I removed it now

**Pull request merged**